lib/minitest/reporters/base_reporter.rb

```diff
@@ -85,12 +85,13 @@ def print(*args)
         io.print(*args)
       end
 
-      def print_info(e, name=true)
-        print "#{e.exception.class.to_s}: " if name
+      def print_info(e, error=true)
         e.message.each_line { |line| print_with_info_padding(line) }
 
-        trace = filter_backtrace(e.backtrace)
-        trace.each { |line| print_with_info_padding(line) }
+        unless error
+          trace = filter_backtrace(e.backtrace)
+          trace.each { |line| print_with_info_padding(line) }
+        end
```

Comment:
I'm not sure I like the fix :( perhaps I'm missing something.
Before the fix the method always prints backtraces and also print name if name/error param is true.
After the change it always prints name and prints backtraces only if name/error is true.
Imho it is too much to fix problem with particular exception (and it is unclear why the new code will work with other exceptions).

As for parameter name: I'd prefer to make it describe itself in print_info's context, imho the best name would be `print_name` (or something similar).  `error` parameter of `print_info` says nothing about its meaning for the method (should I pas true of false) and I have to read `print_info` code to understand what it means.  `name` is not the best param, but `error` doesn't improve the situation thus I see no reasons to change it.